pkg/ebpf/tracer_windows.go

```diff
@@ -45,7 +171,46 @@ func (t *Tracer) getConnections(active []ConnectionStats) ([]ConnectionStats, ui
 
 // GetStats returns a map of statistics about the current tracer's internal state
 func (t *Tracer) GetStats() (map[string]interface{}, error) {
-	return nil, ErrNotImplemented
+	if t.driverHandle == syscall.InvalidHandle {
+		return nil, fmt.Errorf("Problem with handle cannot get stats.")
+	}
+
+	var (
+		bytesReturned uint32
+		stats         C.struct_driver_stats
+		statbuf       = make([]byte, C.sizeof_struct_driver_stats)
+		ioctlcd       = ctl_code(NETWORK_DEVICE_TYPE_CTL_CODE, DDFILTER_IOCTL_GETSTATS, uint32(0), uint32(0))
+	)
+
+	err := syscall.DeviceIoControl(t.driverHandle, ioctlcd, nil, 0, &statbuf[0], uint32(len(statbuf)), &bytesReturned, nil)
+	if err != nil {
+		log.Errorf("Error reading Stats with DeviceIoControl: %v", err)
+	}
+
+	stats = *(*C.struct_driver_stats)(unsafe.Pointer(&statbuf[0]))
+
+	return map[string]interface{}{
+		"driver_total_stats": map[string]int64{
+			"read_calls":             int64(stats.Total.Read_calls),
+			"read_bytes":             int64(stats.Total.Read_bytes),
+			"read_calls_outstanding": int64(stats.Total.Read_calls_outstanding),
+			"read_calls_cancelled":   int64(stats.Total.Read_calls_cancelled),
+			"read_packets_skipped":   int64(stats.Total.Read_packets_skipped),
+			"write_calls":            int64(stats.Total.Write_calls),
+			"write_bytes":            int64(stats.Total.Write_bytes),
+			"ioctl_calls":            int64(stats.Total.Ioctl_calls),
+		},
+		"driver_handle_stats": map[string]int64{
```

Comment:
I'm unsure we want to export these stats since it is on a per-handle basis. That being said, we want to export stats on a per-handle basis, I can create a scheme to do so.